app/src/main/java/org/schabi/newpipe/local/holder/LocalStatisticStreamItemHolder.java

```diff
@@ -110,6 +111,10 @@ public void updateFromItem(final LocalItem localItem,
             itemProgressView.setVisibility(View.GONE);
         }
 
+        if (!DependentPreferenceHelper.getPositionsInListsEnabled(itemProgressView.getContext())) {
+            itemProgressView.setVisibility(View.GONE);
+        }
+
```

Comment:
Remove this and just add the `getPositionsInListsEnabled` condition to the `if` on line 101, for the same reason as above